log/src/main/java/kcl/teamIndexZero/traffic/log/Log_Config.java

```diff
@@ -1,6 +1,6 @@
 package kcl.teamIndexZero.traffic.log;
 
-import kcl.teamIndexZero.traffic.log.fileIO.FileInput;
+import kcl.teamIndexZero.traffic.log.FileIO.FileInput;
```

Comment:
@lexaux When I pulled a fresh version of the project on Kumar's laptop some of the packages you manually renamed to lowercase registered as the old uppercase ones. Weird behaviour.
@kumarawijeet try renaming "kcl.teamIndexZero.traffic.log.**FileIO**.FileInput;" to "kcl.teamIndexZero.traffic.log.**fileIO**.FileInput; and see if it's still ok.